packages/mirinae/src/feedbacks/alert/toast-alert/PToastAlert.vue

```diff
@@ -87,17 +86,16 @@ const state = reactive({
 </script>
 
 <style lang="postcss">
+/* Applying 'important' to resolve the issue where CSS is not being applied due to a change in the timing of the loading of 'vue-notification-group' during build. */
```

Comment:
suggestion: Good comments! But I think it would be better to emphasize it more :)
Also, this is not happened during build time but loading time on the application on the browser. So I suggest change the description about it.

```suggestion
/* CAUTION: Do not remove 'important'.
 * This resolves the issue where CSS is not being applied due to a change in the timing of the loading of original 'vue-notification-group' libarary's css on the application at running time. */
```